akd_local_auditor/src/storage/dynamodb.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! This module comprises AWS DynamoDb bucket READ ONLY access to download and parse
+//! Audit Proofs
+
+use clap::Args;
+
+#[derive(Args, Debug, Clone)]
+pub struct DynamoDbClapSettings {
+    /// The S3 bucket where the audit proofs are stored
+    #[clap(long)]
+    test: String,
+}
```

Comment:
Likely just a side-effect of this still being a WIP, but figured I'd point out that this looks like it has some copy-paste remnants form the world of S3 :)

Reply:
Yeah it's more as a placeholder should it come, and showing how you "might" implement other storage traits. I'll note it